pkg/images/utils/utils.go

```diff
@@ -264,3 +279,35 @@ func FilterSuppressedCVEsNoClone(img *storage.Image) {
 		}
 	}
 }
+
+// IsRedHatImage takes in an image and returns whether it's a Red Hat image.
+//
+// This function is used to determine whether an image is supposed to have been built and signed by Red Hat, for supply
+// chain provenance
+func IsRedHatImage(img *storage.Image) bool {
+	return slices.ContainsFunc(img.GetNames(), IsRedHatImageName)
+}
+
+// IsRedHatImageName takes in an image name and returns whether it corresponds to a Red Hat image
+//
+// This is determined via heuristics, by looking at these images, which are assumed to be "official Red Hat images",
+// and checking where they are hosted:
+//
+//   - All images running in a default openshift cluster
+//   - All images that are referred by PackageManifests in the "redhat-operators" OLM catalog
+//
+// See the description of https://github.com/stackrox/stackrox/pull/15761 for details
+func IsRedHatImageName(imgName *storage.ImageName) bool {
```

Comment:
I think it might be u exported

Reply:
Done: https://github.com/stackrox/stackrox/pull/15761/commits/6fb46d7c8c066a63e68230e4f74485d68f127fac